Source/Fuse.Controls.Native/Interfaces.uno

```diff
@@ -123,6 +123,7 @@ namespace Fuse.Controls.Native
 	public interface IScrollViewHost
 	{
 		float PixelsPerPoint { get; }
+		float2 ContentSize { get; }
```

Comment:
This needs a CHANGLEOG entry... though I suspect this interface should actually be internal.

Reply:
I'll add that. But yeah, everything in that file should have been `internal`

Reply:
Or chnage them to internal and note that in the CHANGELOG.

Reply:
Yeah, I think IScrollViewHost should be made internal.

Reply:
`IScrollViewHost` is used in public `UXConstructor`'s. Cannot easily be made internal

Reply:
I'll do a pass on `Fuse.Control.Native` and make whats need to be internal internal and make a separate PR